pkg/vfsgen/vfsgen_data.go

```diff
@@ -1650,6 +1729,7 @@ var Assets = func() http.FileSystem {
 	}
 	fs["/charts/crds/templates"].(*vfsgen۰DirInfo).entries = []os.FileInfo{
 		fs["/charts/crds/templates/crds.yaml"].(os.FileInfo),
+		fs["/charts/crds/templates/crds.yaml.orig"].(os.FileInfo),
```

Comment:
why crds.yaml.orig?

Reply:
i think these ended up in the compiled-in charts by accident. anyway, the compiled in charts option is about to be removed.